client/app/lib/onboarding.coffee

```diff
@@ -98,8 +98,7 @@ class Step
 
     # Error handler for save() call
     handleSaveError: (err) =>
-        error = Object.values err.errors
-        throw new Error error
+        throw new Error err.error
```

Comment:
I tested the worklfow when password was too short; server error was formated as `{ error: 'mesage', trans: 'message', errors: 'password is too short' }`. Was this changed in your PR?